config/development.json

```diff
@@ -28,6 +28,7 @@
 	"features": {
 		"accept-invite": true,
 		"ad-tracking": false,
+		"brazil-survey-invitation":true,
```

Comment:
Team global has a support rotation next week, which is a little awkward.

I'll discuss it with the team, but I'd love to have the option to launch today.